src/settings_gui.cpp

```diff
@@ -1172,6 +1182,14 @@ void SettingsContainer::Init(byte level)
 	}
 }
 
+/** Resets all settings to their default values */
+void SettingsContainer::ResetAll()
+{
+	for (EntryVector::iterator it = this->entries.begin(); it != this->entries.end(); ++it) {
```

Comment:
Should be possible to do a ranged for I think.

Reply:
Done!